docs/INSTALL.rst

```diff
@@ -16,11 +16,11 @@ Prerequisite Software
 HIL requires a number of packages available through the CentOS RPM
 repositories, as well as the EPEL repository. EPEL can be enabled via::
 
-    yum install epel-release
+    $ yum install epel-release
```

Comment:
oh boy, you went through all the trouble of adding the $ sign everywhere. Sorry I wasn't paying attention that you were taking this up. But if you think about it putting the dollar sign kinda makes it harder to copy and paste it. Earlier, I could triple click and copy the whole line and paste it into a terminal, but with this I would have to remove the dollar sign each time. :(

Don't make this change yet, maybe I copy paste wrong and what you are doing is right. I want to hear from other reviewers or whoever suggested putting the dolla sign there.

Reply:
> Don't make this change yet,

I mean, don't undo your changes yet.

Reply:
hah, thought we had a conclusion on that issue.
@shwsun ping~

Reply:
I don't quite understand the debate. It is convention to have a `$` denote that it is a command rather something that you should put into a file. We need to distinguish things should be in the config file from commands we want users to run. 

@xuhang57 thank you for your time to clean the HIL doc. While you are at it, could you fix [this](https://github.com/xuhang57/hil/blob/62b5246436c4f5a539c3efd30892966bb6ab044c/docs/INSTALL.rst#user-need-to-choose-appropriate-values-for-their-environment) too? It annoys me a while.

Reply:
sure, "user needS" and `export` before each line? That's all right?

Reply:
> I don't quite understand the debate. It is convention to have a $ denote that it is a command rather something that you should put into a file.

I guess I am not a fan of that convention then, for the reasons I mentioned in my comment.

> We need to distinguish things should be in the config file from commands we want users to run.

I don't know, but I feel it's apparent from the context and that the fact that we mention on top of the block of lines if it goes into a file.

In any case, I'll wait to hear what others have to say; but now I am inclined to follow the convention just because that's the convention.

Reply:
@Izhmash @zenhack @SahilTikale @knikolla  anyone want to weigh in?

Reply:
No strong opinion, but my inclination is to include the `$`, as these are meant to be instructions, not a script -- we should be optimizing for understanding, not copy & paste. The `$` is conventional and provides a useful cue to the reader, even if we do mention above that it's a shell command.

Reply:
I am used to seeing `$` when reading documentation.  It helps when you need to quickly scan through a wall of text to piece out what needs to be run in the terminal.